src/aggregations/bucket-aggregations/histogram-aggregation-base.js

```diff
@@ -146,6 +146,7 @@ class HistogramAggregationBase extends BucketAggregationBase {
      * Set's the range/bounds for the histogram aggregation.
      * Useful when you want to include buckets that might be
      * outside the bounds of indexed documents.
+     * NOTE: Only available in Elasticsearch v7.10.0+
```

Comment:
Could you move this under the documentation for hard bounds?

Reply:
Done